docs/guides/openshift/merlininstall.md

```diff
@@ -1,6 +1,28 @@
 
 This guide assumes RedHat OpenShift is running, which will allow for the install of IBM i Merlin.
 
+# Merlin Requirements
+
+IBM i Modernization Engine for Lifecycle Integration can be [installed from either OpenShift web console](docs/guides/openshift/merlininstall.md) or CLI (`oc` command).
+
+## Supported product versions
+
+See [Supported product versions](./guides/platform/Install_IBM_i_Modernization_Engine_for_Lifecycle_Integration.md).
```

Comment:
[Supported product versions]
should be updated to
[Supported product versions and OpenShift requirements]

Reply:
Done